scripts/run_benchmark.py

```diff
@@ -0,0 +1,24 @@
+#!/bin/env/python
+"""Script to run the benchmark on a vector database."""
+
+import argparse
+from pathlib import Path
+
+from inatinqperf.benchmark.benchmark import Benchmarker
+
+
+def main() -> None:
+    """Entry point."""
+    p = argparse.ArgumentParser(description="VectorDB-agnostic benchmark")
```

Comment:
Use `typer` rather than `argparse`.

Reply:
That makes sense, however we have publicly stated in multiple forums that our solutions use `typer` rather than `argparse`. Also, consider that we are now asking people to learn 2 arg parsing tools (`typer` and `argparse`).

Reply:
Oh I see. That's a solid reason for me. :)